src/Domain/Tags.hpp

```diff
@@ -122,6 +123,33 @@ struct ElementMap : db::SimpleTag {
   using type = ::ElementMap<VolumeDim, TargetFrame>;
 };
 
+/// \ingroup DataBoxTagsGroup
+/// \ingroup ComputationalDomainGroup
+/// Base tag to retrieve the centers of objects in the domain corresponding to
+/// the `ObjectLabel`.
+template <ObjectLabel Label>
+struct GridCenter : db::BaseTag {};
```

Comment:
Please define these tags in a new file in Domain/Tags/. The Tags.hpp file has become quite cluttered and we've moved to a Tags/ subdirectory. Helps with recompilations too.

Reply:
ping on this (move to own file). Maybe name `ObjectCenter` or `BinaryObjectCenter`? That the coords are in the grid frame is now implied by the data type. Same for the simple tag (maybe just `ExcisionCenter`?)